src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -48,15 +48,26 @@ private static CSharpCompilation CreateCompilationWithFunctionPointers(CSharpTes
             return CreateCompilation(source, references: references, options: options ?? TestOptions.UnsafeReleaseDll, parseOptions: TestOptions.Regular9, targetFramework: targetFramework ?? TargetFramework.Net50);
         }
 
+        private static CSharpCompilation CreateCompilationWithFunctionPointers(CSharpTestSource source, bool includeUnmanagedCallersOnly, CSharpCompilationOptions? options = null)
+        {
+            var references = includeUnmanagedCallersOnly
+                ? TargetFrameworkUtil.GetReferences(TargetFramework.Net50)
+                : TargetFrameworkUtil.GetReferencesWithout(TargetFramework.Net50, "System.Runtime.InteropServices.dll");
+            return CreateCompilation(source, references: references, options: options ?? TestOptions.UnsafeReleaseDll, parseOptions: TestOptions.Regular9, targetFramework: TargetFramework.Empty);
```

Comment:
>parseOptions: TestOptions.Regular9 [](http://example.com/codeflow?start=119&length=34)

Not specific to this PR, but are we testing function pointers with `-langversion:latest`, or are most tests using `-langversion:9`?